src/main/kotlin/day03/Hero.kt

```
@@ -1,26 +1,24 @@
     fun usingSkills() {
-
-
     }
 }
+
+class Sword(
+    val name: String,
+    val damage: Int,
+){
+
+}
```

detekt (1.23.7)

[warning] The class or object Sword is empty.

(detekt.empty-blocks.EmptyClassBlock)